data/context/standard_test.go

```diff
@@ -152,6 +171,16 @@ var _ = Describe("Standard", func() {
 				Expect(child).ToNot(BeNil())
 			})
 
+			It("Logger returns a logger", func() {
+				Expect(child.Logger()).ToNot(BeNil())
```

Comment:
Yes, but we really don't care what the concrete type is, just that it implements the interface. We know that it implements the interface because if it didn't Go wouldn't compile it.  Therefore, the way I see it, we don't ever need to check something like this. The only time we'd need to check a type is if we need to ensure a concrete type. For example, when parsing some raw upload type data, the `Parse` function returns `data.Datum`. Then, we'd want to validate that it actually returned a concrete `upload.Upload` type. Something more specific that `data.Datum`.

Does that make sense?